pkg/utils/predicates.go

```diff
@@ -107,3 +108,14 @@ func TolerationsTolerateTaintsWithFilter(tolerations []v1.Toleration, taints []v
 
 	return true
 }
+
+// GetNodeFitFromStrategyParams gets nodeFit configuration variable from the given StrategyParameters.
+// It will return false by default.
+func GetNodeFitFromStrategyParams(params *api.StrategyParameters) bool {
```

Comment:
I'm not sure if this is the right place to put this helper function, maybe we can add a new `utils.go` file to `descheduler/pkg/descheduler/strategies/` to place it(and we can add more related helper functions to this file)?

Reply:
Good point, I wasn't really sure where to put that one. I've created `utils.go` and placed this function there.